extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -104,6 +103,11 @@ class AmpStickyAd extends AMP.BaseElement {
       this.deferMutate(() => {
         toggle(this.element, true);
         this.viewport_.addToFixedLayer(this.element);
+        // Add border-bottom to the body to compensate space that was taken
+        // by sticky ad, so no content would be blocked by sticky ad unit.
+        const borderBottom = this.element./*OK*/offsetHeight;
```

Comment:
I agree. The API name should be `updatePaddingBottom`. Whether inside the `Viewport` it's implemented as a padding or a border - that's up to the implementation.